.env.example

```
@@ -17,10 +17,24 @@ VECTOR_DB_HOST=localhost
 VECTOR_DB_PORT=8765
 VECTOR_DB_CREDENTIALS=
 
-# LLM (expert, generic, guidance, ingest plugins)
+# LLM — provider-agnostic configuration
+# Supported providers: mistral, openai, anthropic
+LLM_PROVIDER=mistral
+LLM_API_KEY=
+LLM_MODEL=                     # Default: mistral-large-latest / gpt-4o / claude-sonnet-4-6
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)